.github/workflows/build-test.yml

```diff
@@ -0,0 +1,31 @@
+name: Node CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: saucelabs connect
+        uses: saucelabs/sauce-connect-action@v0.0.1
+        with:
+          user: $SAUCE_USERNAME
+          key: $SAUCE_ACCESS_KEY
+      - name: npm install, build, and test
+        run: |
+          npm ci
+          npm run build --if-present
+          COVERAGE=true npm test
+          ./node_modules/coveralls/bin/coveralls.js < ./coverage/lcov.info;
+        env:
+          CI: true
```

Comment:
Adding [this](https://github.com/babel/preset-modules/blob/dff636d75ca8760a4882c7b65252fe443546c242/.github/workflows/main.yml#L26-L32) as the last action and removing the current SauceLabs setup should work:

```
    - name: saucelabs
      env:
        CI: true
        SAUCE_USERNAME: ${{ secrets.SAUCE_USERNAME }}
        SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY }}
      run: |
        npm run test:karma
```

We will just need to change the `test` script to skip karma tests in CI.